### DISCUSSION TITLE
How can I show errors, escaped and killed Mutants and syntax errors??

### DISCUSSION THREAD

**Discussion by steevanb** [infection/infection#1564] [Q&A] [answered]
Hi,

How can i show details about `covered mutants were not detected` and `errors were encountered`?

I've tried with `-v`, `-vv` and `-vvv`, i've not found anything about it with `--help` or in https://infection.github.io/.

The only thing is the json log can be created, but i'm not sure i will have expected informations here and how to have it directly in CLI?

![2021-09-04_17h23_07](https://user-images.githubusercontent.com/5063383/132099601-6f6879e1-6c42-457f-8da1-34d1deae01ac.png)

**Comment by maks-rafalko** [comment, marked as answer]
Hi,

* to output escaped mutants to `stdout` (your terminal), please use [`--show-mutations`](https://infection.github.io/guide/command-line-options.html#show-mutations-or-s), or just its alias `-s`
* to use human readable log file, please use [`text`](https://infection.github.io/guide/usage.html) logger - in `infection.json` add `logs.text`
* information added to `text` log file is controlled by [`--log-verbosity`](https://infection.github.io/guide/command-line-options.html#log-verbosity)

if you will use `--log-verbosity=all`, it will add logs about killed and errored mutants

> The verbosity of the log file, `all` - this mode will add `“Killed mutants”` into log file and add additional information, `default` - normal mode will skip `“Killed mutants”` section in the log file, `none` - which will disable logging to files.

so the final solution is

`infection.json`
```json
{
    "logs": {"text": "infection.log"}
}
```

and execute Infection with

`infection -s --log-verbosity=all`

If you want to also log PHPUnit's output, as well as CLI commands that are executed internally by Infection, please use [`--debug`](https://infection.github.io/guide/command-line-options.html#debug)

if you think we can improve anything with the docs, or with the help output - please advise.

**Comment by steevanb** [comment]
Thanks it's perfect!
